app/repositories/v2/ListRepository.scala

```diff
@@ -57,6 +58,11 @@ class ListRepository @Inject() (
 
   override lazy val requiresTtlIndex: Boolean = config.isP5TtlEnabled
 
+  private def otherError(error: String): Left[OtherError, Nothing] = {
+    logger.warn(error)
+    Left(OtherError(error))
+  }
```

Comment:
A todo for the future would be to make more specific errors for certain scenarios rather than using OtherError. This is fine though :)